src/mongoc/mongoc-topology-description.c

```diff
@@ -1370,6 +1370,46 @@ gSDAMTransitionTable[MONGOC_SERVER_DESCRIPTION_TYPES][MONGOC_TOPOLOGY_DESCRIPTIO
 /*
  *--------------------------------------------------------------------------
  *
+ * _mongoc_topology_description_type_string --
+ *
+ *      Get this topology's type, one of the types defined in the Server
+ *      Discovery And Monitoring Spec, as a string.
+ *
+ *      NOTE: this method should only be called while holding the mutex on
+ *      the owning topology object.
+ *
+ * Returns:
+ *       A string.
+ *
+ * Side effects:
+ *       None.
+ *
+ *--------------------------------------------------------------------------
+ */
+static const char *
+_mongoc_topology_description_type_string (mongoc_topology_description_t *topology)
```

Comment:
The only usecase for this function and mongoc_server_description_type_string () is the one TRACE() entry?
I don't think thats useful enough, and prefer keeping the trace lines as-is I guess